index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+content. Any of these additional checks might be integrated into future
+versions of this specification.
+      </p>
+
+      <section class="normative">
+        <h3>Verification</h3>
+
+        <p>
+This section contains an algorithm that <a>conforming verifier
+implementations</a> can run when verifying a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
+<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
+inputs, and produces an object (<var>verified</var>) that contains the
+following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>verified</var> to an empty object.
+          </li>
+          <li>
+Ensure that the securing mechanism has properly protected the
+<var>conforming document</var> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>securingMechanismVerified</var> to the result of executing the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> with <var>inputBytes</var> and <var>mediaType</var> passed as
+inputs to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>mediaType</var>,
+<var>warnings</var>, and <var>errors</var> in <var>verified</var> to their
+respective properties in <var>securingMechanismVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a document that cannot be considered to be
+secured. Return <var>verified</var>, which will contain warning and error
+information related to the verification failure.
+              </li>
+            </ol>
+          </li>
+          <li>
+Ensure that the <var>verified</var>.<var>document</var> is a
+<a>conforming document</a> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>dataModelVerified</var> to the result of executing the
+<a href="#verify-data-model">data model verification algorithm</a>
+with <var>document</var> passed as input to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>warnings</var>, and
+<var>errors</var> in <var>verified</var> to their respective properties in
+<var>dataModelVerified</var>.
```

Comment:
status is validation, not verification... checking for well formed conforming document is also validation.

I recognize data integrity does things differently, but if we are to have normative guidance regarding verification in the core data model, it can't be based on splitting the difference between securing mechanisms.